Sources/PaymentButtons/PaymentButtonFont.swift

```diff
@@ -1,12 +1,15 @@
 import UIKit
 
 enum PaymentButtonFont {
-    /// The primary font to be employed in the UI for non special circumstances
-    public static let primaryFont: UIFont = .systemFont(ofSize: UIFont.systemFontSize)
+    /// The primary font used for non special circumstances
```

Comment:
I know this docstring existed before but it's a bit confusing to me, mainly the "non special circumstances" bit. Maybe we can just update to "The primary font used". Thoughts?

Reply:
Thanks! Yea it was a bit wordy and "The primary font used" would be enough